### PR TITLE
BTT SKR-SE-BX (STM32H743IIT6 ARM Cortex M7) and BIQU_BX_TFT70

### PR DESCRIPTION
**Pull request by bigtreetech**
### Description
* STM32H743IIT6 ARM Cortex-M7 run in 400Mhz
* LTDC + onborad SDRAM + onboard QSPI Flash direct drive 7 inch (1024 * 600) TFT screen (`TFT_COLOR_UI` is now available.  Other UI will be compatible and added later)

### CODE REVIEW

File: Marlin/src/HAL/STM32/MarlinSPI.cpp [hunk 1]
```diff
@@ -19,7 +19,7 @@
  * along with this program.  If not, see <https://www.gnu.org/licenses/>.
  *
  */
-#if defined(ARDUINO_ARCH_STM32) && !defined(STM32GENERIC)
+#if defined(ARDUINO_ARCH_STM32) && !defined(STM32GENERIC) && !defined(STM32H7xx)
```
Comment (rhapsodyv) [inline Marlin/src/HAL/STM32/MarlinSPI.cpp:22]: We need to add the spi code for H7, to enable this class.

**Comment by rhapsodyv** [comment]
As this is the only H7 board in Marlin, it would be nice to add a test for this too. It may give some guarantee that others changes don't break it.

**Comment by bigtreetech** [comment]
> As this is the only H7 board in Marlin, it would be nice to add a test for this too. It may give some guarantee that others changes don't break it.

https://github.com/MarlinFirmware/Marlin/pull/21536/checks?check_run_id=2267009949
Hello, Do you know why a `Permission denied` error here?

**Comment by ellensp** [comment]
Marlin/buildroot/tests/BTT_SKR_SE_BX is not marked as being executable
eg if you look at https://github.com/bigtreetech/Marlin/blob/btt-bx-se/buildroot/tests/BIGTREE_SKR_PRO at the top it says "Executable File  33 lines (28 sloc)  1.11 KB"
but looking at https://github.com/bigtreetech/Marlin/blob/btt-bx-se/buildroot/tests/BTT_SKR_SE_BX at the top it says "18 lines (15 sloc)  284 Bytes"  not marked as executable

**Comment by ellensp** [comment]
```
git add --chmod=+x -- buildroot/tests/BTT_SKR_SE_BX 
git commit -m"Executable!"
```
should fix it.

**Comment by bigtreetech** [comment]
> ```
> git add --chmod=+x -- buildroot/tests/BTT_SKR_SE_BX 
> git commit -m"Executable!"
> ```
> 
> should fix it.

It's ok now.  Thank you very much!

**Comment by ellensp** [comment]
https://github.com/MarlinFirmware/Marlin/commit/98d6c751d3484b1a3d30cefaa692b4f993d81abd breaks CI

**Comment by InsanityAutomation** [comment]
I had actually done most of this over the weekend in a local branch, so glad to see it here!! Ill get some testing in this week!

Edit : I see the ext_ui code isnt included here, is that intended to be still here to drive the touch mode?

**Comment by thisiskeithb** [comment]
> Edit : I see the ext_ui code isnt included here, is that intended to be still here to drive the touch mode?

It’ll be added later, so no touch for now.

**Comment by bigtreetech** [comment]
> I had actually done most of this over the weekend in a local branch, so glad to see it here!! Ill get some testing in this week!
> 
> Edit : I see the ext_ui code isnt included here, is that intended to be still here to drive the touch mode?

Hello, My plan is to create a new PR to add `ext_ui` after this PR merged. to ensure the `ext_ui` of BTT UI style  sustainable update. we need to be adapted BTT ext_ui according to Marlin's framework, which is a big task.
So I want to add a simple and available PR to Marlin first. Add then  add other features step by step, different functions submit different PR rather than putting all in one PR, which makes it easy to code review and fault backtracking.

**Comment by InsanityAutomation** [comment]
Sounds good! If I get some time while I'm stuck in a hotel next week I'll go over that code a bit. I think I can sneak 1 printer on the trip, still debating what to bring lol

**Pull request merged**